src/BloomExe/ImageProcessing/ImageUtils.cs

```diff
@@ -51,7 +51,7 @@ Icon    B96B3CB5
 		/// If the image has a filename, replaces any file with the same name.
 		/// </summary>
 		/// <returns>The name of the file, now in the book's folder.</returns>
-		public static string ProcessAndSaveImageIntoFolder(PalasoImage imageInfo, string bookFolderPath)
```

Comment:
If the caller tells us that the image file is actually the same file we started out with, then we can build the output path without worrying about overwriting another file.  Otherwise, we have to ensure a unique name for the output file.